docs/en/enterprise-edition/content-collections/runtime-security/configure/permissions.adoc

```diff
@@ -81,3 +81,17 @@ The following table shows the required permissions for agentless scanning.
 https://main\--prisma-cloud-docs-website\--hlxsites.hlx.live/en/compute-edition/assets/compute-feature-wise-permissions?sheet=azure-agentless-30-03
 |===
 
+=== Agentless Scanning Resource Group Permissions
```

Comment:
@jenjoe22 Did you also update the PCCE page?
I don't see the file listed.
https://docs.prismacloud.io/en/compute-edition/32/admin-guide/configure/permissions

Reply:
@rcaballeromx Omri confirmed that the update does not need to be made for on-prem versions.